docs/storage/custom-storage-domains.md

```diff
@@ -0,0 +1,62 @@
+﻿
+
+# Custom Storage API Domains
+
+Upon creation of a Fleek.co Storage bucket, users are assigned a default API URL that looks something like: 'Storageapi.fleek.one` + bucket ID.
+
+With Custom Storage API Domains, we're increasing the uptime, availability, and accessibility of these storage buckets by allowing existing Fleek.co storage users to add their own custom domains to map to their Fleek storage buckets.
+
+!!! info
+
+    When you deploy on Fleek your site will have an IPFS hash and future automatic deployments will also update the IPFS hash of your site. Your site comes with a CDN, free SSL certificate, and a Fleek preview URL. You can add Custom Domains, ENS Domains, Configure Deploy Previews, and more.
+
+## Add Custom Domain
+
+Navigate to the storage tab, denoted by a `new` prompt and found in the vertical navigation bar of the Fleek.co app.
+
+![](imgs/add-domain1.png)
+
+#### 1.  Click `Add a Custom Domain`
+
+#### 2.  Enter the Domain you want to map to your storage bucket -- make sure this is a domain you own.
+
+#### 3.  Click `Check DNS Configuration` and copy the URL that is in the format of 'bucket-id.b-cdn.net`, in order to let the custom domain know the location of your storage bucket.

```

Comment:
This is not the case. I think this might be from an outdated doc because this was caught and updated in the doc that was sent last week.

Reply:
Added a modified text in the last commit based on this feedback.

Reply:
We should still mention that whats in the Points to field is what they need to set their DNS to.